daml-lf/engine/src/test/scala/com/digitalasset/daml/lf/engine/EngineTest.scala

```diff
@@ -1067,6 +1067,9 @@ object EngineTest {
   private implicit def toParty(s: String): Party =
     Party.assertFromString(s)
 
+  private implicit def toContractId(s: String): ContractIdString =
+    LedgerString.assertFromString(s)
```

Comment:
```suggestion
    ContractIdString.assertFromString(s)
```

As a consequence of the proposed aliases for `LedgerString`, and similarly for other direct references to `LedgerString` in order to call methods.

Reply:
mostly done. I have some more cleanup to do. I will fix the remaining ones (EventId, WorkflowId, ApplicationId ...)  in an upcoming PR.